flow/material-ui-core.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+/* eslint-disable */
+
+declare module '@material-ui/core' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/styles' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Button' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Dialog' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Checkbox' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Form' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Switch' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Progress' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/green' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/styles/colorManipulator' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/IconButton' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Tooltip' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/CssBaseline' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/Paper' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/colors/grey' {
+    declare module.exports: any;
+}
+
+declare module '@material-ui/core/colors/blue' {
+    declare module.exports: any;
```

Comment:
Does `any` give us any benefits when `type` checking at all?

Reply:
You're right, it doesn't. The reason behind the declaration is to let flow know a module exists under '@material/core/colors/blue' and get rid of the "cannot resolve module" error in the import statement.

Reply:
Ok, just wondering. I'm a Flow newbie so bear with me. :dancing_men: